google/cloud/spanner_dbapi/__init__.py

```diff
@@ -93,7 +97,7 @@ def connect(
     if not database.exists():
         raise ValueError("database '%s' does not exist." % database_id)
 
-    return Connection(instance, database)
+    return Connection(instance, database, default_pool)
```

Comment:
While in standard interface, we're putting the default pool. However, one will be able to set his own pool by initiating a `Connection` object manually.

Reply:
That sounds good.

Could we add an exit handler that automatically closes the default pool (if it exists and is open) when the application is shutdown?

Reply:
This can be done with [atexit](https://docs.python.org/3/library/atexit.html) package. That's the question to ask googlers I suppose, as I'm not sure if there are any underwater rocks.

Reply:
I don't see any reason not to use atexit here -- just that the handler should include a short timeout so we don't keep the process open if we don't get a timely response from spanner.